### PR TITLE
updated watershed.py

### PR DESCRIPTION
**Pull request by chengwill97**
- updated the radical.pilot API syntax in watershed.py
- updated to use argparse instead of sys.args
- removed RP profiler from watershed.py

pls accept

### CODE REVIEW

File: watershed/Chocolate/watershed.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    try:
+        
+        # Add a Pilot Manager
+        pmgr = rp.PilotManager(session=session)
+
+        # Provide the Pilot Description
+        #
+        pdesc = rp.ComputePilotDescription()
+        pdesc.resource  = resource
+        #pdesc.project = 'TG-MCB090174'      # for Stampede/Wrangler
```
Comment (iparask) [inline watershed/Chocolate/watershed.py]: Project should be setup when running remotely

File: watershed/Chocolate/watershed_lines.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    print('Input Arguments')
+    pp([   ['read_from        ' , read_from],
+           ['read_to   		  ' , read_to],
+           ['bright_background' , bright_background]
+       ])
+if args.verbosity >= 1:
+    print 'Arguments are valid'
+
+# Use for testing on Will's local linux VM
+path = '/Users/WillC/Documents/Rutgers/Research/RADICAL/watershed/midas/watershed/Chocolate'
```
Comment (iparask) [inline watershed/Chocolate/watershed_lines.py]: This has to be an argument

File: watershed/Chocolate/watershed_lines.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+import argparse
+import pprint
+pp = pprint.PrettyPrinter().pprint
+
+
+#-------------------------------------------------------------------------------
+parser = argparse.ArgumentParser()
+
+# pilot cores
```
Comment (iparask) [inline watershed/Chocolate/watershed_lines.py]: Change the comments

**Review by iparask — CHANGES_REQUESTED** [review]

File: watershed/Chocolate/watershed.py [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+                        help='set image background brightness')
+    # report name
+    parser.add_argument('-r','--report',        type=str, default='watershed_report',
+                        help='report name')
+    # resource
+    parser.add_argument('-R','--resource',      type=str, default='local.localhost_anaconda',
+                        help='specify the resource to use')
+    # verbosity
+    parser.add_argument('-v', '--verbosity',    action='count', default=0,
+                        help='increase output verbosity')
```
Comment (iparask) [inline watershed/Chocolate/watershed.py]: Add arguments for project, queue and runtime.

File: watershed/Vanilla/watershed.py [hunk 5]
```diff
@@ -125,12 +114,12 @@
...
         # Provide the Pilot Description
         #
-        pdesc = rp.ComputePilotDescription()
+        pdesc           = rp.ComputePilotDescription()
         pdesc.resource  = resource
         pdesc.project   = project
         pdesc.runtime   = walltime
         pdesc.cores     = pilot_cores
-	pdesc.queue 	= queue
+	    pdesc.queue 	= queue
```
Comment (iparask) [inline watershed/Vanilla/watershed.py]: Why these extra spaces here? The indentation does not look correct. Can you make sure you are using 4 spaces for each indentation level?

**Review by iparask — CHANGES_REQUESTED** [review]

File: watershed/Vanilla/watershed.py (hunk 5 above)
Comment (chengwill97) [reply watershed/Vanilla/watershed.py]: That's really  odd. It doesn't show up as that on my computer :\. I'll try to find out what's going on

Comment (chengwill97) [reply watershed/Vanilla/watershed.py]: Okay I fixed it

**Review by iparask — APPROVED** [review]

**Pull request merged**